_security-plugin/access-control/document-level-security.md

```diff
@@ -124,3 +124,26 @@ PUT _plugins/_security/api/roles/abac
   }]
 }
 ```
+## Use term-level lookup queries (TLQs) with DLS 
+
+You can perform term-level lookup queries (TLQs) with document-level security (DLS) using either of two modes: adaptive or filter-level. The default mode is adaptive, where OpenSearch automatically switches between Lucene level or filter level mode depending on whether or not there is a term-level lookup query. DLS queries without TLQs are executed at the Lucene level mode, whereas DLS queries with TLQs are executed on filter level mode.
```

Comment:
Hyphenate "filter-level" and "Lucene-level" when used as compound modifiers preceding the noun, but do not hyphenate when they follow the noun or stand on their own. Change to "two modes: adaptive or filter level", "between Lucene-level or filter-level mode", "executed at Lucene-level mode" (remove "the" preceding "Lucene"), and "executed on filter-level mode." Please use "TLQ" for the second instance of "term-level lookup query." In the third sentence, it looks like the prepositions are either inconsistent or incorrect. Should it be "executed in Lucene-level mode" and "executed in filter-level mode"?

Reply:
thanks so much!